lesson_05/homework_05.py

```diff
@@ -1,29 +1,54 @@
 small_list = [3, 1, 4, 5, 2, 5, 3]
 big_list = [3, 5, -2, -1, -3, 0, 1, 4, 5, 2]
 # task 1. Знайдіть всі унікальні елементи в списку small_list
+small_list = set(small_list)
+print(f"Унікальні елементи", small_list)
 
 # task 2. Знайдіть середнє арифметичне всіх елементів у списку small_list
+average = sum(small_list) // len(small_list)
+print(f"Середнє арифметичне {average}")
 
 # task 3. Перевірте, чи є в списку big_list дублікати
+duble = set(big_list)
+duble_1 = len(duble)
+print(f"Дублікати:", duble_1)
 
 
 base_dict = {'contry':'Ukraine', 'continent': 'Europe', 'size': 123}
 add_dict = {"a":1, "b":2, "c":2, "d":3, 'size': 12}
```

Comment:
```suggestion
add_dict = {"a":1, "b":2, "c":2, "d":333,'size': 12}
```
спробуйте на такому варіанті